tests/cases/fourslash/quickInfoCanBeTruncated.ts

```diff
@@ -0,0 +1,520 @@

```

Comment:
re your question about multiple truncations: they're really bad. The length is so long that multiple truncations don't appear near each other, and the truncation start looking like some kind of Typescript-specific syntax.

That said, I have no idea how to make things better besides some vague ideas that sound like a lot of work.